docs/quantum/guides/circuits.rst

```diff
@@ -181,7 +190,14 @@ To build up the mixing circuit, we need to put parametrized RX gate on each of t
 .. note::
   ``orquestra-quantum`` comes with a `built-in utility function <https://github.com/zapatacomputing/orquestra-quantum/blob/main/src/orquestra/quantum/circuits/_generators.py>`_ that puts down a layer of the same single-qubit gate acting on all qubits.
 
-The output of that looks like ``mixing_circ: Circuit(operations=[RX(gamma_0)(0), RX(gamma_1)(1), RX(gamma_2)(2)], n_qubits=3)``. Again, for now don't worry about the ``gamma`` parameters in there, that will be addressed in the :ref:`symbolic gates <symbolic_gates>` section.
+The output of that looks like this:
+
+.. code-block:: text
+  
+  ic| mixing_circ: Circuit(operations=[RX(gamma_0)(0), RX(gamma_1)(1), RX(gamma_2)(2)], n_qubits=3)
+  Circuit(operations=[RX(gamma_0)(0), RX(gamma_1)(1), RX(gamma_2)(2)], n_qubits=3)
```

Comment:
looks like the output is duplicated. suggestion: leave the icecream printout, 

```suggestion
```

Reply:
right, looks like I overlooked this